src/modules/NtpClient.rb

```diff
@@ -944,6 +944,13 @@ def Import(settings)
           next deep_copy(p)
         end
       end
+
+      restricts = settings["restricts"] || []
```

Comment:
I miss a comment here, at `Import` you convert `restricts` to a Hash, at `Export` back from Hash. Why is this needed? It's not obvious from the diff...